src/angular-app/bellows/core/filters.ts

```diff
@@ -20,10 +19,10 @@ export type RelativeTimeFilterFunction = (timestamp?: string, timeFormat?: strin
 
 export function RelativeTimeFilter(): RelativeTimeFilterFunction {
   return (timestamp?: string, timeFormat?: string): string => {
-    // see http://momentjs.com/docs/
-    const timeAgo = moment(timestamp, timeFormat);
-    if (timeAgo.isValid()) {
-      return timeAgo.fromNow();
+    const date = new Date(timestamp);
+    const dateNow = new Date();
+    if (isValid(date) && isValid(dateNow)) {
+      return formatDistance(date, dateNow, { addSuffix: true });
```

Comment:
This can be simplified by using [the `formatDistanceToNow` function](https://date-fns.org/v2.23.0/docs/formatDistanceToNow), and then you don't need `dateNow`.

Reply:
Also, [beware of parsing dates with the `new Date(someString)` constructor](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/Date#parameters)! The Mozilla documentation warns against it:

> **Note:** Parsing of date strings with the `Date` constructor (and `Date.parse()`, which works the same way) is strongly discouraged due to browser differences and inconsistencies.
> * Support for [RFC 2822](https://datatracker.ietf.org/doc/html/rfc2822) format strings is by convention only.
> * Support for ISO 8601 formats differs in that date-only strings (e.g. `"1970-01-01"`) are treated as UTC, not local.

Proof that this can have very surprising results (Firefox on Linux):

![image](https://user-images.githubusercontent.com/90762/128980441-99908dc1-2caa-4b72-a369-a07b47947a20.png)

I ran the following snippet in Node to verify this behavior, and sure enough:

```js
import { isValid, formatDistanceToNow } from 'date-fns';

function test(timestamp) {
    const date = new Date(timestamp);
    if (isValid(date)) {
      console.log(date);
      return formatDistanceToNow(date, { addSuffix: true });
    }
    return ''
}

console.log(test('2021-07-15'))
console.log(test('2021/07/15'))
```

Result:

```plaintext
2021-07-15T00:00:00.000Z
27 days ago
2021-07-14T17:00:00.000Z
28 days ago
```

Since the date strings coming into the filter are from MongoDB and should be in ISO format (if they aren't, it's a bug) then it's probably best to use date-fns' [parseISO](https://date-fns.org/v2.23.0/docs/parseISO) function instead.